src/app.js

```diff
@@ -1,17 +1,36 @@
 import express from 'express'
 import userRouter from './router/userRouter'
-import handleValidationError from './error/handleValidationError'
-import handleError from './error/errorHandler'
+import internalRouter from './router/internalRouter'
+import validationErrorHandler from './midelware/validationErrorsMiddelware'
```

Comment:
Folder should be called `middleware` instead of `midelware`

Reply:
Fixed